libraries/AC_PID/AC_PID.cpp

```diff
@@ -261,7 +261,19 @@ float AC_PID::get_ff()
 
 void AC_PID::reset_I()
 {
-    _integrator = 0;
```

Comment:
Isn't this going to change the behaviour of all AC_PID objects that use reset I?  Is that the intention?

Reply:
Yes.  From discussion with @lthall, he thought this was acceptable.  Most of the time that multi's are using the reset, they aren't spinning props so this shouldn't affect them.  It is really noticeable in heli's.  You may have already noticed this in Loiter with the current Leaky I and ILMI implementation.

Reply:
Yep, happy with this.